fhir-search/src/main/java/org/linuxforhealth/fhir/search/SearchConstants.java

```diff
@@ -94,6 +94,12 @@ private SearchConstants() {
 
     // _source
     public static final String SOURCE = "_source";
+    
+    // _firstId the resource Id of the first resource of the next search result.
+    public static final String FIRST_ID = "_firstId";
+
+    // _lastId the resource Id of the last resource of the previous search result.
+    public static final String LAST_ID = "_lastId";
```

Comment:
as search parameters, I find it easier to think of these as expectations for the page being requested
```suggestion
    // _firstId the expected resource Id of the first resource on the requested page of search results.
    public static final String FIRST_ID = "_firstId";

    // _lastId the expected resource Id of the last resource on the requested page of search results.
    public static final String LAST_ID = "_lastId";
```

I guess it really just depends which context you are thinking about this from... the current descriptions are certainly correct if we're thinking about it from the server's perspective while constructing the previous and next links.

Reply:
This has been fixed as suggested.